src/App.tsx

```diff
@@ -1,26 +1,64 @@
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
+import { getTodos } from './api';
```

Comment:
Ensure that the `getTodos` function is correctly implemented in the `./api` module. This function should return a promise that resolves with the list of todos from the server.